### PR TITLE
Map even more tests to web-features

### PR DESCRIPTION
**Pull request by foolip**
The list of tests for nesting are from here:
https://github.com/web-platform-tests/interop/issues/420#issuecomment-1755268425

### CODE REVIEW

**Review by fserb — APPROVED** [review]
canvas LGTM

File: css/css-grid/animation/WEB_FEATURES.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+features:
+- name: grid-animation
```
Comment (flackr) [inline css/css-grid/animation/WEB_FEATURES.yml:2]: Is the name meant to reference a spec? If so this is testing part of the css-grid spec: https://drafts.csswg.org/css-grid/#track-sizing

**Review by bashi — APPROVED** [review]
webtransport LGTM

**Review by dbaron — APPROVED** [review]
css/* LGTM, modulo one comment below

File: css/css-flexbox/WEB_FEATURES.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+features:
+- name: flexbox
+  files: "**"
```
[the remark this answers is not in the document]
Comment (foolip) [reply css/css-flexbox/WEB_FEATURES.yml:3]: Excellent point! I made this PR before we had flexbox-gap in web-features, but now I should figure out how to go about mapping that...

[the remark this answers is not in the document]
Comment (foolip) [reply css/css-flexbox/WEB_FEATURES.yml:3]: I'll need wildcards to select the flexbox-gap tests, which depends on https://github.com/web-platform-tests/wpt/pull/43403, so I'm going to break this out into a separate PR.

[the remark this answers is not in the document]
Comment (foolip) [reply css/css-flexbox/WEB_FEATURES.yml:3]: https://github.com/web-platform-tests/wpt/pull/43702

**Pull request merged**